feathr_project/test/test_azure_synapse_e2e.py

```diff
@@ -42,6 +46,57 @@ def test_feathr_online_store():
         assert res['265'][1] != None
 
 
+def test_feathr_online_store_non_agg_features():
+    """
+    Test FeathrClient() online_get_features and batch_get can get data correctly.
+    """
+    test_workspace_dir = Path(__file__).parent.resolve() / "test_user_workspace"
+    os.chdir(test_workspace_dir)
+    client = FeathrClient()
+
+
+    # client.materialize_features()
+    # # just assume the job is successful without validating the actual result in Redis. Might need to consolidate
+    # # this part with the test_feathr_online_store test case
+    # client.wait_job_to_finish(timeout_sec=600)
+    res = client.online_get_features('nycTaxiDemoFeature', '111', ['f_gen_trip_distance', 'f_gen_is_long_trip_distance',
```

Comment:
Where are these 'f1', 'f2', 'f3', 'f4', 'f5', 'f6' features from?  Overall, better description of this test case would be better.

Reply:
this is in the test config file.